### ISSUE TITLE
DiagnosticSuppressor doesn't detect tuple deconstruction assignments

### ISSUE THREAD

**Issue by manfred-brands** [nunit/nunit.analyzers#499]
The diagnostic suppressor detects assignments in `SetUp` methods.
It doesn't however detect tuple deconstruction assignments:

```csharp
(m_Name, m_Description) = CreateInstances()
```

It does work if manually deconstructed:

```csharp
(string Name, string Description) instances = CreateInstances();
m_Name = instances.Name;
m_Description= instances.Description;
```

We need to detect tuple deconstruction assignments.
